include/z64state.h

```diff
@@ -0,0 +1,39 @@
+#ifndef Z64STATE_H
+#define Z64STATE_H
+
+#include "ultra64/ultratypes.h"
+#include "gfx.h"
```

Comment:
Why does this header need `gfx.h`?

Reply:
for `GraphicsContext`, but it could indeed be a forward struct declaration instead. I'll change it

Reply:
Yeah, that was exactly my thought, a forward declaration seems better in this case

Reply:
Bump